purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -268,17 +274,18 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             identityManager.currentAppUserID to deviceCache.cachedOfferings
         }
         if (cachedOfferings == null) {
-            debugLog("No cached offerings, fetching")
+            log(LogIntent.DEBUG_INFO, OfferingStrings.NO_CACHED_OFFERINGS)
             fetchAndCacheOfferings(appUserID, state.appInBackground, listener)
         } else {
-            debugLog("Vending offerings from cache")
+            log(LogIntent.DEBUG_INFO, OfferingStrings.VENDING_OFFERINGS_CACHE)
             dispatch {
                 listener.onReceived(cachedOfferings)
             }
             state.appInBackground.let { appInBackground ->
                 if (deviceCache.isOfferingsCacheStale(appInBackground)) {
-                    debugLog("Offerings cache is stale, updating cache")
+                    log(LogIntent.DEBUG_INFO, OfferingStrings.OFFERINGS_STALE_BACKGROUND)
```

Comment:
`appInBackground` would determine if this is in being called in background or foreground

Reply:
@vegaro wouldn't `if (deviceCache.isOfferingsCacheStale(appInBackground)` already check that the app is in background?

Reply:
`isOfferingsCacheStale` checks if the caches is stale, depending on the app being in background or not. If the caches are stale, the if condition will still be true even if the app is in foreground.